inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -1105,12 +1165,15 @@ else if (identifierKBObj.get() instanceof KBInstance) {
             tupleQuery.setBinding("pSUBCLASS", aKB.getSubclassIri());
             tupleQuery.setBinding("pLABEL", aKB.getLabelIri());
             tupleQuery.setBinding("pDESCRIPTION", aKB.getDescriptionIri());
+            tupleQuery.setBinding("pSUBPROPERTY", aKB.getSubPropertyIri());
             tupleQuery.setIncludeInferred(false);
 
             return evaluateListQuery(aKB, tupleQuery, false, aAll, "s");
         });
 
-        resultList.sort(Comparator.comparing(KBObject::getUiLabel));
+        if (resultList.size() > 1) {
```

Comment:
Why extra check - if the list is empty or contains a single element, then the sort is a NOP - performance hit should be minimal.